listenbrainz_spark/stats/artist.py

```diff
@@ -2,9 +2,73 @@
 import time
 from listenbrainz_spark import config
 from listenbrainz_spark.stats import run_query
+from dateutil.relativedelta import relativedelta
+from datetime import datetime 
+from collections import defaultdict
 
+def get_releases():
+    result = run_query("""
+        SELECT artist_msid
+             , release_name
+             , release_mbid
+             , release_msid
+             , count(release_msid) as cnt
+          FROM listen
+      GROUP BY artist_msid, release_name, release_mbid, release_msid
+      ORDER BY cnt DESC
+    """)
+    result.show()
+    releases = defaultdict(list)
+    for row in result.collect():
+        releases[row.artist_msid].append({
+            'release_name' : row.release_name,
+            'release_msid' : row.release_msid,
+            'release_mbid' : row.release_mbid,
+            'count' : row.cnt,
+        })
+    return releases
+
+def get_recordings():
+    result = run_query("""
+        SELECT artist_msid
+             , track_name
+             , recording_mbid
+             , recording_msid
+             , count(recording_msid) as cnt
+          FROM listen
```

Comment:
In the other PR, we pass around the name of the temporary table. I know I am the one who started the name listen but we should have better names and pass them around too (in consistency with the other parts of the codebase)